scripts/gen_operator_csv.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+#
+# Generate an operator bundle for publishing to OLM. Copies appropriate files
+# into a directory, and composes the ClusterServiceVersion which needs bits and
+# pieces of our rbac and deployment files.
+#
+# Usage ./scripts/gen_operator_csv.py OUTPUT_DIR PREVIOUS_VERSION IMAGE_NAME
+# Example: scripts/gen_operator_csv.py tmp 0.1 quay.io/redhat/dedicated-admin-operator:latest
```

Comment:
```suggestion
# Example: scripts/gen_operator_csv.py tmp 0.1 quay.io/redhat/dedicated-admin-operator:latest
# If there is no previous version, specify __undefined__ instead of 0.1, ex:
# scripts/gen_operator_csv.py tmp __undefined__ quay.io/redhat/dedicated-admin-operator:latest
```

Give an example for the suggested change below